.docker/images/nginx/tests/src/AcsfRedirectsTest.php

```diff
@@ -34,20 +21,33 @@ public function setUp(): void {
     `docker-compose exec nginx tar -czvf /app/sites/g/files/net123/f/private/backups/backup.sql.tar.gz --files-from /dev/null`;
   }
 
+  /**
+   * A list of common ACSF paths.
+   *
+   * @return array
+   *   A list of paths.
+   */
+  public function providerRedirectPath() {
+    return [
+      ['/sites/g/files/net12409/themes/site/mysite/autotest.jpg'],
+      ['/sites/g/files/net1234/f/autotest.jpg'],
+    ];
+  }
+
   /**
    * Ensure that ACSF paths correctly redirect to new locations.
    *
-   * @dataProvider provideAcsfPaths
+   * @dataProvider providerRedirectPath
    */
-  public function testRedirect($acsf_path): void {
-    $headers = \get_curl_headers($acsf_path);
+  public function testRedirectPath($path) {
```

Comment:
no need to call it `$acsf_path` - we do not have any other paths so just `$path` is sufficient.